gammapy/data/observations.py

```diff
@@ -342,6 +382,61 @@ def select_time(self, time_interval):
         obs.obs_filter = new_obs_filter
         return obs
 
+    @classmethod
+    def from_file(cls, event_file, irf_file):
```

Comment:
Can we have a default for `irf_file=None` that would mean "Look for the IRFs in the `event_file`"?

Reply:
I think for simulations it's also useful to make the event file optional as well. This e.g. will allow us to finally get rid of https://github.com/gammapy/gammapy/blob/master/gammapy/irf/io.py#L76

Reply:
As it's a convenience function anyway, one could also simplify the logic and just allow for one filename. In this case it means to just read everything that's contained in the file. If the data is stored in two separate files, the following works as well, just with one additional line of code:

```python3
obs = Observation.read("my-dl3-irfs.fits")
obs.events = EventList.read("my-obs-events.fits")
```

Reply:
This is done now.